src/lev-page/index.js

```diff
@@ -13,7 +13,7 @@ const GlobalFont = createGlobalStyle`
 
 const LevPage = ({ topnav, footer, footerContent, title, font, children, ...props }) => {
   const Header = <LevTopNav { ...topnav } />;
-  const Footer = <LevFooter { ...footer } >{footerContent}</LevFooter>;
+  const Footer = (footer || footerContent) ? <LevFooter { ...footer } >{footerContent}</LevFooter> : null;
```

Comment:
Why not:
```js
const Footer = (footer || footerContent) && <LevFooter { ...footer } >{footerContent}</LevFooter>;
```

Reply:
Because react and `false` don't work the same way as regular JS - I think you have to use `null`

Reply:
Okay.